apps/mobile/app/(tabs)/_layout.tsx

```diff
@@ -2,8 +2,14 @@ import { Tabs } from 'expo-router'
 import React from 'react'
 import { TabBarIcon } from '@/src/components/navigation/TabBarIcon'
 import { Navbar as AssetsNavbar } from '@/src/features/Assets/components/Navbar/Navbar'
+import useNotifications from '@/src/hooks/useNotifications'
 
 export default function TabLayout() {
+  /**
+   * Enables notifications for the app if the user has enabled them
+   */
+  useNotifications()
```

Comment:
Is this the best place to check permissions? 
should we have a NotificationProvider wrapped around the whole navigation, so we check the permissions first thing the user starts the app?